cmd/assetsvc/postgresql_utils.go

```diff
@@ -92,11 +101,21 @@ func (m *postgresAssetManager) getPaginatedChartList(namespace, repo string, pag
 	return charts, 1, nil
 }
 
-func (m *postgresAssetManager) getChart(namespace, chartID string) (models.Chart, error) {
+func (m *postgresAssetManager) getChart(namespace, chartIDUnescaped string) (models.Chart, error) {
 	var chart models.ChartIconString
+	chartID, _ := url.PathUnescape(chartIDUnescaped)
```

Comment:
I'm guessing you meant to change this, but we can't ignore errors like this, it needs to be:
```
chartID, err := url.PathUnescape(chartIDUnescaped)
if err != nil {
    return models.Chart{}, err
}
```

Reply:
Maybe this is explained elsewhere but why don't we store the chartID in the database escaped? So we don't need the extra escaping/unescaping.